src/appleseed/renderer/modeling/bsdf/oslbsdf.cpp

```diff
@@ -34,6 +34,7 @@
 #include "renderer/global/globaltypes.h"
 #include "renderer/kernel/shading/closures.h"
 #include "renderer/kernel/shading/shadingpoint.h"
+#include "renderer/modeling/bsdf/alsurfacelayerbrdf.h"
 #include "renderer/modeling/bsdf/bsdf.h"
 #include "renderer/modeling/bsdf/bsdffactoryregistrar.h"
```

Comment:
Fixed and rebased.

About max_value(), value can be a color if fresnel mode is metallic or if the reflectance is a color. What alSurface does (if I read the code correctly) is:

F * reflectance * microfacet(...) + (1.0 - max_value(F * reflectance)) * substrate()